apps/admin-portal/src/components/applications/wizard/wizard-protocol-oauth-settings.tsx

```diff
@@ -94,8 +102,37 @@ export const WizardOAuthProtocolSettings: FunctionComponent<OAuthProtocolSetting
                                                 }
                                             });
                                         } }
-                                        value={ callBackURL && buildCallBackURLWithSeparator(callBackURL) }
+                                        value={ OIDCdata &&
+                                        buildCallBackURLWithSeparator(OIDCdata.callbackURLs?.toString()) }
                                     />
+                                    <Hint>
+                                        After the authentication, we will only redirect to the above callback
+                                        URLs.
+                                        You can specify multiple URLs by separating them using a comma.
+                                    </Hint>
+
+                                </Grid.Column>
+                            </Grid.Row>
+                            <Grid.Row columns={ 1 }>
+                                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 14 }>
+                                    <Field
+                                        name="publicClients"
+                                        label=""
+                                        required={ false }
+                                        requiredErrorMessage="this is needed"
+                                        type="checkbox"
+                                        value={ templateData &&
+                                        templateData.inboundProtocolConfiguration.oidc.publicClient ? ["supportPublicClients"] : [] }
```

Comment:
Fix character length
```suggestion
                                        templateData.inboundProtocolConfiguration.oidc.publicClient ?
                                            ["supportPublicClients"] : [] }
```